servlet/build.xml

```diff
@@ -52,6 +53,38 @@
       <!-- additional dependencies (used by scripts). -->
     </gwt.jar>
   </target>
+  <target name="-servlet-jakarta" description="Packages this project into a jar for use with modern jakarta apis">
+    <mkdir dir="${gwt.build.lib}" />
+    <gwt.jar destfile="${gwt.build.lib}/gwt-servlet-jakarta.jar">
+      <!-- Rebased dependencies go in gwt-servlet too -->
+      <zipfileset src="${gwt.tools.lib}/guava/guava-19.0/guava-19.0-rebased.jar" />
+      <zipfileset src="${gwt.tools.lib}/jscomp/20160315/sourcemap-rebased.jar" />
+      <zipfileset src="${gwt.tools.lib}/json/android-sdk-19.1/json-android-rebased.jar" />
+      <zipfileset src="${gwt.tools.lib}/streamhtmlparser/streamhtmlparser-jsilver-r10/streamhtmlparser-jsilver-r10-1.5-rebased.jar" />
+      <zipfileset src="${gwt.tools.lib}/protobuf/protobuf-2.5.0/protobuf-java-rebased-2.5.0.jar" />
```

Comment:
if https://github.com/gwtproject/gwt/pull/9785 lands, this will need the same change.